src/partition.c

```diff
@@ -1555,7 +1553,7 @@ static void pick_scotch(int nodeID, struct space *s, int nregions,
           failed++;
         }
         if (weights_e[k] < 1) {
-          message("Used edge weight out of range: %" PRIDX, weights_e[k]);
+          message("Used edge weight out of range: %" "I64d", (long long)weights_e[k]);
```

Comment:
Does the I64d still cause a problem in debug mode?

Reply:
No. All 4 debug-enabled binaries [mpm,s32,s64,slocal] build OK with those format specifiers and the casting in those lines as per commit https://github.com/UCL/swiftsim/pull/27/commits/bb4d93b2da297d9bd82faea79f166ef0fe891574